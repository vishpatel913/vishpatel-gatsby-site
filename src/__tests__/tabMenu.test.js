import React from "react";

import renderWithTheme from "../utils/test-renderer";
import TabMenu from "../components/tabMenu";

describe("TabMenu", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(<TabMenu />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
