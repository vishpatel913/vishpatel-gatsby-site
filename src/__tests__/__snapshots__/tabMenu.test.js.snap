// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabMenu renders correctly 1`] = `
.c0 {
  display: block;
}

.c1 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  height: auto;
  margin: 0.5rem;
  margin-bottom: 1.5rem;
  background: #fff;
  border: solid 1px #E0E0E0;
  border-radius: 4px;
}

.c2 {
  display: inherit;
  margin: 0;
  border-right: solid 1px #E0E0E0;
}

.c2:last-child {
  border: none;
}

.c3 {
  font-size: 12px;
  font-weight: lighter;
  padding: 0.5rem 1rem;
  color: #595959;
}

.c3:hover,
.c3.active {
  color: #512DA8;
  background: #f7f7f7;
}

@media (max-width:768px) {
  .c0 {
    text-align: center;
  }
}

@media (max-width:768px) {
  .c1 {
    margin: 1rem 1rem 0;
  }
}

@media (max-width:400px) {
  .c3 {
    padding: 0.5rem;
  }
}

<div
  className="c0"
>
  <ul
    className="c1"
  >
    <li
      className="c2"
    >
      <a
        className="c3"
        href="/work"
      >
        All
      </a>
    </li>
    <li
      className="c2"
    >
      <a
        className="c3"
        href="/work/design"
      >
        Design
      </a>
    </li>
    <li
      className="c2"
    >
      <a
        className="c3"
        href="/work/development"
      >
        Development
      </a>
    </li>
    <li
      className="c2"
    >
      <a
        className="c3"
        href="/work/photography"
      >
        Photography
      </a>
    </li>
  </ul>
</div>
`;
